src/contract.ts

```diff
@@ -63,86 +66,19 @@ export class Contract {
      * @param {Session} session - The session object to use to sign the transaction.
      * @return {Promise<TransactResult>} A promise that resolves with the transaction data.
      */
-    async call(
-        name: NameType,
-        data: ABISerializableObject | {[key: string]: any},
-        session: Session
-    ): Promise<TransactResult> {
-        const action: Action = Action.from({
-            account: this.account,
-            name,
-            authorization: [],
-            data,
-        })
-
-        // Trigger the transaction using the session kit
-        return session.transact({action})
-    }
-
-    /**
-     * Gets all the tables for the contract.
-     *
-     * @return {Promise<Table[]>} A promise that resolves with all the tables for the contract.
-     */
-    async getTables(): Promise<Table[]> {
-        const abi = await this.getAbi()
-
-        return abi.tables.map((table) => {
-            return new Table({
-                contract: this,
-                name: table.name,
-            })
-        })
-    }
-
-    /**
-     * Gets a specific table for the contract.
-     *
-     * @param {NameType} name - The name of the table.
-     * @return {Promise<Table>} A promise that resolves with the specified table.
-     */
-    async getTable(name: NameType): Promise<Table> {
-        const tables = await this.getTables()
-
-        const table = tables.find((table) => table.name.equals(name))
-
-        if (!table) {
-            throw new Error(`No table found with name ${name}`)
-        }
-
-        return table
-    }
-
-    /**
-     * Gets the ABI for the contract.
-     *
-     * @return {Promise<ABI.Def>} A promise that resolves with the ABI for the contract.
-     */
-    async getAbi(): Promise<ABI.Def> {
```

Comment:
If I recall, we didn't actually come to any decision on whether or not we wanted to keep the lazy load approach for the ABI. Personally, I'm still a fan of the lazy load approach.

Reply:
We can always add it back in. For now though I'd rather see how far we can get without it.